commands/team_test.go

```diff
@@ -108,3 +108,250 @@ func (s *MmctlUnitTestSuite) TestCreateTeamCmd() {
 		s.Require().Len(printer.GetLines(), 0)
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestRemoveUserCmd() {
+	teamArg := "example-team-id"
+	userArg := "example-user-id"
+	s.Run("Remove users from team, without args returns an error", func() {
+		printer.Clean()
+
+		err := removeUsersCmdF(s.client, &cobra.Command{}, []string{})
+		s.Require().Equal(err, errors.New("Not enough arguments."))
+		s.Require().Len(printer.GetLines(), 0)
+	})
+
+	s.Run("Remove users from team, with one arg returns an error", func() {
+		printer.Clean()
+
+		err := removeUsersCmdF(s.client, &cobra.Command{}, []string{teamArg})
+		s.Require().Equal(err, errors.New("Not enough arguments."))
+		s.Require().Len(printer.GetLines(), 0)
+	})
+
+	s.Run("Remove users from team, team not found, returns an error", func() {
+		printer.Clean()
+
+		s.client.
+			EXPECT().
+			GetTeam(teamArg, "").
+			Return(nil, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetTeamByName(teamArg, "").
+			Return(nil, &model.Response{Error: nil}).
+			Times(1)
+
+		err := removeUsersCmdF(s.client, &cobra.Command{}, []string{teamArg, userArg})
+		s.Require().Equal(err, errors.New("Unable to find team '"+teamArg+"'"))
+		s.Require().Len(printer.GetLines(), 0)
+	})
+
+	s.Run("Remove users from team, user not found error", func() {
+		printer.Clean()
+		mockTeam := &model.Team{Id: teamArg}
+		mockUser := &model.User{Id: userArg}
+
+		s.client.
+			EXPECT().
+			GetTeam(teamArg, "").
+			Return(mockTeam, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetUserByEmail(mockUser.Id, "").
+			Return(nil, nil).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetUserByUsername(mockUser.Id, "").
+			Return(nil, nil).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetUser(mockUser.Id, "").
+			Return(nil, nil).
+			Times(1)
+
+		err := removeUsersCmdF(s.client, &cobra.Command{}, []string{teamArg, mockUser.Id})
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 0)
+		s.Require().Len(printer.GetErrorLines(), 1)
+		s.Require().Equal(printer.GetErrorLines()[0], "Can't find user '"+userArg+"'")
+	})
+
+	s.Run("Remove user by email get team by name, no error", func() {
```

Comment:
"Remove users from team by email and get team by name should not return an error"